fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/FromItem.java

```diff
@@ -1,22 +1,84 @@
 /*
- * (C) Copyright IBM Corp. 2019
+ * (C) Copyright IBM Corp. 2019,2021
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
 package com.ibm.fhir.database.utils.query;
 
+import com.ibm.fhir.database.utils.query.expression.StatementRenderer;
+
 /**
- * An entry in the FROM clause
+ * An element in the FROM clause
```

Comment:
Could you give a couple examples? 

FROM FHIRDATA.T1 T1, FHIRDATA.T2 T2? something like that?

Reply:
ok